### ISSUE TITLE
FAQ: Will it read passport NFC chip?

### ISSUE THREAD

**Issue by urbien** [hansemannn/iOS11-NFC-Example#12]
On Android that was possible for a while, but if it is finally possible on iOS 11.
Note that some passports, like US, needs to be open for NFC chip to be read. Also, to read the data on passport's chip you need to provide input information found in MRZ zone.

**Comment by hansemannn** [comment]
Unfortunately, no. iOS 11 only allows to read `NDEF NFC-Tags` as stated in the readme already.

**Comment by urbien** [comment]
indeed, I found a more detailed answer here: https://www.readid.com/demo - section on IOS support.

> Do you support iOS / iPhone?
> The short answer is not yet. The long answer is that we were very happy when Apple introduced the iPhone 6 with NFC, and some iPads also have NFC. Starting iOS 11 iOS also provides the possible for third-party app developer to use this NFC capabilities by providing an API. Unfortunately, for now this API is limited to so-called NDEF tags, while to read ePassport chip we need to be able to read so-called ISO 14443 tags (a and b). We hope and think that Apple will make this possible at some point in the not-so-distant future.
> 
> We do have an iOS implementation for scanning the MRZ (machine readable zone), but not a public demo app for this. And we can use external NFC readers connected via Bluetooth or cable to an iPhone or iPad, contact if you have a use case for this.
>

**Comment by hansemannn** [comment]
Great background, thank you! Reopening as an FAQ question.
